src/components/BatchVoting/DefaultOptions/OptionWrapper.tsx

```
@@ -0,0 +1,722 @@
+import AbstainOptions from './AbstainOptions';
+import { IOptionsWrapper } from '../types';
+import Image from 'next/image';
+import { SegmentedValue } from 'antd/es/segmented';
+
+export const getConvictionVoteOptions = (CONVICTIONS: [number, number][], proposalType: ProposalType, api: ApiPromise | undefined, apiReady: boolean, network: string) => {
+	if ([ProposalType.REFERENDUM_V2, ProposalType.FELLOWSHIP_REFERENDUMS].includes(proposalType) && ![AllNetworks.COLLECTIVES, AllNetworks.WESTENDCOLLECTIVES].includes(network)) {
+		if (api && apiReady) {
+			const res = api.consts.convictionVoting.voteLockingPeriod;
+			const num = res.toJSON();
+			const days = blockToDays(num, network);
+			if (days && !isNaN(Number(days))) {
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)